README.md

```diff
@@ -12,8 +12,8 @@ This repository contains the Puppet modules for WSO2 Micro Integrator.
     ```bash
     sudo puppet module install puppetlabs-java
     ```
-5. Download and update wso2mi-4.2.0 pack. Then copy it to the `<puppet_environment>/modules/micro_integrator/files` as `wso2mi-4.2.0.zip` directory.
-6. [Optional] Download and update wso2mi-dashboard-4.2.0 pack. Then copy it to the `<puppet_environment>/modules/mi_dashboard/files` as `wso2mi-dashboard-4.2.0.zip` directory.
+5. Download and update wso2mi-4.3.0 pack. Then copy it to the `<puppet_environment>/modules/micro_integrator/files` as `wso2mi-4.3.0.zip` directory.
+6. [Optional] Download and update wso2-integration-control-plane-1.1.0 pack. Then copy it to the `<puppet_environment>/modules/integration_control_plane/files` as `integration-control-plane-1.1.0.zip` directory.
```

Comment:
IMO, it is better to have ICP 1.0.0 with MI 4.3.0 and ICP 1.1.0 with MI 4.4.0. So we will have puppet resources for both ICP 1.0.0 and 1.1.0 in two branches. If anyone wants puppet resources for ICP 1.0.0, they can come to 4.3.0 branch. WDYT?

Reply:
I had the impression mi 4.3.0 and 4.4.0 both will have icp 1.1.0. No issue in changing.